frontend/src/components/pages/chat/components/messageList/MessageList.tsx

```diff
@@ -9,21 +10,30 @@ const MY_USER_ID = 'apple';
 
 export const MessageList: React.FC = () => {
   const [messages, setMessages] = useState<Array<any>>([]);
+  const { dialogId } = useParams<any>();
 
   useEffect(() => {
-    setMessages([...tempMessages]);
-    // id
-  }, []);
+    setMessages([]);
+    (async () => {
+      const messages = await getChatMessages(dialogId);
+      setMessages(messages);
+    })();
+  }, [dialogId]);
 
-  //scroll to bottom
+  //TODO: second useEffect for user data
+
+  // third useEffect for sockets for new messages
+
+  //scroll to bottom on each render
   const divRef = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
     divRef.current!.scrollIntoView();
   });
 
-  const renderMessages = () =>
+  const renderMessages = () => {
```

Comment:
with this change function returns undefined
If you were using TS for function return types - this error will be shown immediately